test/integration/query-interface/removeColumn.test.js

```diff
@@ -54,7 +54,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         expect(table).to.not.have.property('firstName');
       });
 
-      it('should be able to remove a column without default value', async function () {
+      it('should be able to remove a column without a default value', async function () {
```

Comment:
consider moving this non-controversial change to another PR as a way to lighten this (pretty big) one